internal/controller/cloudsyncmgr/cloudsync.go

```diff
@@ -36,7 +36,7 @@ const (
 type CloudSync interface {
 	InitiateCloudSync(isCloudSet string) error
 	//implemented by external REST API
-	RequestCloudSyncConf(host string, clientID string, message mqttmgr.Message, topic string) string
+	RequestCloudSyncPublish(host string, clientID string, message mqttmgr.Message, topic string) string
```

Comment:
Simply, what about `Publish`?

Reply:
Do you mean just Publish(), I thought of this idea, but then thought would be confusing for nay neww developer to understand by just Publish and hence this name.

Reply:
However this function is in the CloudSync. What about `RequestPublish` instead?

Reply:
seems good